include/device_plugin_container.h.in

```diff
@@ -9,10 +9,30 @@ class DeviceImpl;
 class PluginImplBase;
 ${FORWARD_DECLARATION_STRING}
 
+/**
```

Comment:
@mrpollo This file is not being parsed by doxygen.

Reply:
@mrpollo You need to do `make default install` and then create the docs for the `install/include` directory:

```
install
├── include
│   └── dronecore
│       ├── action.h
│       ├── device.h
│       ├── device_plugin_container.h
│       ├── dronecore.h
│       ├── info.h
│       ├── logging.h
│       ├── mission.h
│       ├── mission_item.h
│       ├── offboard.h
│       ├── plugin_base.h
│       └── telemetry.h
└── lib
    └── libdronecore.a
```

Reply:
Changed in 2c23813.